### PR TITLE
Added provider blacklist policy

### PR DESCRIPTION
**Pull request by rtfee**

### CODE REVIEW

**Comment by rtfee** [comment]
@maratkomarov FYI all values listed in the test are fake.

**Comment by maratkomarov** [comment]
@petroprotsakh I'm sure we should include this policy somewhere into github actions yml

**Comment by rtfee** [comment]
@petroprotsakh would you mind reviewing and merging?

File: providers/blacklist_provider.rego [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package terraform
+
+import input.tfplan as tfplan
+
+# Blacklisted Terraform providers
+not_allowed_provider = [
+  "azurerm"
```
Comment (petroprotsakh) [inline providers/blacklist_provider.rego:7]: @rtfee jfyi, interesting fact: starting from Terraform v0.13 provider_name in plan json will be stored as fully qualified source name (e.g. `registry.terraform.io/hashicorp/azurerm`). We'll handle this in SCALRCORE-16889.

**Review by petroprotsakh — APPROVED** [review]

**Pull request merged**